### DISCUSSION TITLE
Docs for Demucs GUI

### DISCUSSION THREAD

**Discussion by CarlGao4** [CarlGao4/Demucs-Gui#23] [Announcements]
I've just finished the development of v1.0a1 and am preparing to create a document for this project. However, I have no idea where to start. Can anybody help me with this? Feel free to write anything related to this document.

**Comment by doublegsoft** [comment]
i have no idea to use this software as well. after loading model, it hangs and no any other buttons to click

**Reply by CarlGao4** [reply]
If you are loading remote models (you will see it in the info area), it will download the models first and you can do nothing during this period of time

**Comment by ysy-888** [comment]
I don't know how to use this either. I'm on macOS. I click Load and then the box gets greyed out and on the bottom there is a message that's something like "Model loaded within 0.5s". I have no idea how to load the audio from here.

**Reply by CarlGao4** [reply]
1.0a1 for macOS (x86_64) has a bug that can't run on Apple Silicon Mac with Rosette2

**Comment by salsantana** [comment]
This is by far the "tightest and rightest" stem extraction tool I've tried. I've much to learn about the GitHub messaging GUI, and really the whole realm of forums for that matter. Can't program my way out of a wet paper bag but very much enjoyed "Structured Program Design and Logic" course in college. I am here to deliver accolades, though.
I compose backing tracks (the sober never-late portable band) and often borrow classical Latin percussion phrases for my rhythm sections. I was absolutely elated when I heard how well your software separated a quartet's phrases...... near enough to excellent for me :)
Many thanks and best wishes for your projects!
Rich

**Comment by DavidMurphy1088** [comment]
I have spent 30 minutes trying to run 1.3.2 and dont have the slighest idea of how it should work. I guess I add files to the file queue and do "start separation". Then the progress bar moves to completed and then absolutely nothing happens. It then jumps to the "mixer" tab and says "waiting for input". But what input???? 
No idea where the output is placed. The UI is completely confusing and this has been a 100% waste of time and very annoying 👹

**Reply by L0rdVetinari** [reply]
Hey, you can come into my Discord and then I could explain it to you. https://discord.gg/XwF4xRrJ
